modules/packages/LinearAlgebra.chpl

```diff
@@ -1886,6 +1886,35 @@ module Sparse {
   /* Transpose CSR matrix */
   proc _array.T where isCSArr(this) { return transpose(this); }
 
+  /* Transpose Sparse domain */
```

Comment:
Since the `where` clauses don't show up in the docs, how about we `pragma "no doc"` these implementations and modify the documented sparse transpose to say something like:

>Supports CSR and COO domains.